### PR TITLE
Prepare 2.6.8 release

### PR DESCRIPTION
**Pull request by j0k3r**
Latest release of 2023 with some bug fix and site configs up to date (huge thanks to @HolgerAusB)

### CODE REVIEW

**Review by yguedidi — DISMISSED** [review]
Approved but I think it's worth waiting for the redirection fix to be merged before release
So the milestone is complete :)
Or maybe should I rebase the fix after this is merged to mention it in the changelog?

**Comment by yguedidi** [comment]
@j0k3r #7140 is merged, this PR can be updated :)

**Comment by j0k3r** [comment]
Rebased to include latest changes from 2.6 and also update deps (with latest site config files)

**Review by yguedidi — APPROVED** [review]

**Pull request merged**
